test/test_examples_3d.jl

```diff
@@ -34,8 +34,9 @@ isdir(outdir) && rm(outdir, recursive=true)
   # Lattice-Boltzmann
   include("test_examples_3d_lbm.jl")
 
-  # Structured mesh
-  include("test_examples_3d_structured.jl")
+  # Curved mesh
+  # Not yet implemented
+  # include("test_examples_3d_curved.jl")
```

Comment:
We changed the data structure (faces instead of `coordinates_min/max`, `metric_terms`, `inverse_jacobian` with different size). We didn't want to make this PR bigger than necessary, and adapt the 3D stuff in a new PR.

Reply:
OK.